src/amoc_api_scenarios_handler.erl

```diff
@@ -128,21 +130,11 @@ to_json(Req0, State) ->
 from_json(Req, State) ->
     case get_vars_from_body(Req) of
         {ok, {ModuleName, ModuleSource}, Req2} ->
-            ScenarioPath = "scenarios/" ++ erlang:binary_to_list(ModuleName),
-            file:write_file(
-              ScenarioPath ++ ".erl",
-              ModuleSource,
-              [write]
-             ),
-            Result = compile_and_load_scenario(
-                       ModuleName,
-                       ScenarioPath),
-            ResultBody = case Result of
-                             ok -> Result;
-                             {error, Errors, _Warnings} ->
-                                 R = io_lib:format("~p", [Errors]),
-                                 erlang:list_to_bitstring(lists:flatten(R))
-                         end,
+            %%Nodes = [amoc_slave:node_name(Node) || Node <- amoc_config:get(hosts, [])],
+            Nodes = [node()],
```

Comment:
The list of nodes consists only of local nodes. Do we want to extend it by other nodes amoc nodes? In the `amoc_dist` there is not exproted function `amoc_nodes`. Maybe we can export it and use it here?

Reply:
I was hesitating what to do.
I think the best would be to export `amoc_nodes` and have one place where we will control how we want to calculate it.